### ISSUE TITLE
Adding author name in original script

### ISSUE THREAD

**Issue by chikiulo** [acl-org/acl-anthology#4045]
Hi,
I am proposing this feature as one of the NAACL 2025 D&I co-chairs. This is also in coordination with ACL 2025 D&I co-chairs.

As per initial discussion with @mjpost, I am creating this issue on github for us to track progress. 

=====================================
Here is the justification for the feature.

When author names in non-Latin languages are transliterated into Latin scripts, ambiguity is introduced due to the fact that 
a) Latin scripts do not handle tones in tonal languages and 
b) there are languages with more homonyms. 
See the huge number of issues related to crashed names resolving in ACL anthology for the extent of the problem. (https://github.com/acl-org/acl-anthology/issues?q=is%3Aissue+is%3Aopen+%22The+author+page+wrongly+conflates+different+people+with+the+same+name.%22) 

Adding the names in original script to the transliterated name is a way to resolve who’s who in the academic world. Some publication venues in other fields of science have already made it into their official policy to include author names in their original scripts for this reason. See https://journals.aps.org/PhysRevLett.99.230001. It would be embarrassing that *ACL and relative conferences are not part of the pioneers in doing that.

**Comment by mbollmann** [comment]
Thanks for making this issue to track progress!

From a technical perspective, this is already supported, but currently only used in the [CCL proceedings](https://aclanthology.org/venues/ccl/). Name variants in a different script appear in brackets after the transliterated form, like so:

![Screenshot of parts of the CCL 2020 proceedings](https://github.com/user-attachments/assets/4e80b6f1-d415-40f1-8a16-7d8314c01132)

This information is currently annotated on a per-paper basis, so for example, while we see in the screenshot that there’s an author "Yu Wang" with the original name 王昱, this variant does not currently appear prominently on [Yu Wang’s author page](https://aclanthology.org/people/y/yu-wang/) (which is also a good illustration for name clashes, as this page conflates several different Yu Wang’s right now ...).

Bottom line: The technical basis for this feature already exists, we’d “just” need to actually obtain and ingest this information from *ACL conferences (and then maybe improve a bit how this information is used and presented).

**Comment by chikiulo** [comment]
Yes, this is also what my discussion with @mjpost led to. And he believes ARR is already collecting name variants via open review. The  next step include updating aclpub2 to extract that information (we may need to also update one of the name variant fields for name in the original script explicitly).

The more complex problem could be venues (e.g.workshops) that still use softconf for submission.

**Comment by chikiulo** [comment]
Should I create an issue at https://github.com/rycolab/aclpub2 to move this forward?

**Comment by chikiulo** [comment]
ping! is there anything I could help to move this forward?
I understand this enchancement may not be in-time for NAACL or even ACL this year but I would still like to see this become available in a close future.

**Comment by mjpost** [comment]
There are a number of issues here that need to be resolved:
1. Representation in the Anthology. We already do this, but it is done at the paper level, rather than at the author level. This is in part because we largely _infer_ the set of people in the Anthology by creating a large set of surface names, and then merging and resolving the two types of ambiguities with the `name_variants.yaml` file. What we'd like to do is move to an explicit representation of people as first-order objects (#623) . The barrier is that this is a big project.
2. There are two separate ingestion methods: (i) Softconf/ACLPUB and (ii) Open Review/aclpub2. In ACLPUB, we can handle name variants, but it's kind of hacky, and recently broke (#3117), presumably due to an upgraded pybtex, which breaks the way we were ingesting the variants (as a duplicate "authors" field).
3. We could add this feature to aclpub2. I suggest creating an issue. However, it will take someone to implement it. You'll need to get a field added to Open Review profiles. OR profiles currently allow multiple names, but (a) I'm not sure if aclpub2 ingests them and (b) they are not named, so I am not sure how we will identify the native-script variant.

**Comment by mbollmann** [comment]
> I am not sure how we will identify the native-script variant.
 
That at least could be done via checking the characters against
http://www.unicode.org/Public/UNIDATA/Scripts.txt

... but it's of course the smallest of the issues you mention.

**Comment by chikiulo** [comment]
If the solution cannot be in-place in a long while because of its scale, what would be the consequences if authors start adding their original names on the pdf only?

I have been doing it for myself for two years and I didn't receive any objections from PCs or organizers. Is this something we could start encouraging authors to do?

**Comment by mbollmann** [comment]
> If the solution cannot be in-place in a long while because of its scale, what would be the consequences if authors start adding their original names on the pdf only?

We don't currently parse PDFs in any way, so if the information is only in the PDF, it is highly uncertain if it will ever appear on the Anthology. It would create even more work to extract this information from PDFs.

**Comment by weissenh** [comment]
It just occurred to me that this issue is not linked in any way to a similar one by the same submitter from about one and a half year earlier: #2576 , so I'd like to have it documented here.

As a side note: Today, the ACL Anthology has switched to a new author representation taking into account ORCID information if available (cf. [this news post](https://aclanthology.org/posts/2026-01-26-new-author-system/) and the now closed issue on Author name URLs already linked above). This should help to disambiguate authors with the same name.

**Comment by nschneid** [comment]
To echo the above, we are now pushing for all authors to submit ORCIDs with their papers to disambiguate them as authors.

Alternate-script variants are represented in the paper metadata when submitted by the venue. While OpenReview, for example, has the ability to record orthographic variants of a name on author profile pages, I don't know if it allows script-variants in the author list of a paper. So I think the work will need to be done upstream of the Anthology, e.g. in aclpub2. Would be happy to see more venues taking advantage of this!
